luma/core/multiimage.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Richard Hull and contributors
+# See LICENSE.rst for details.
+
+from PIL import Image, ImageDraw 
+from luma.core import mixin 
+
+
+class MultiImage(mixin.capabilities):
+    '''
+    Renders multiple images on a background
+    '''
+
+    image_id = 0
+
+    def __init__(self, device, width, height):
+        '''
+        Instantiates a new MultiImage
+        '''
+        self.capabilities(width, height, rotate=0, mode=device.mode)
+        self._device = device
+        self._background_image = Image.new(self.mode, self.size)
+        self.images = {}
+
+    def get_size(self):
+        return self._device.size
+
+    def add_image(self, im, xy=(0, 0), offs=(0, 0)):
```

Comment:
`__setattr__` instead?

Reply:
I'd prefer to keep using the add / remove construct. This way, there is no tight coupling between attribute name and the image, which allows e.g. adding a predefined list of images.